client/ayon_core/hosts/houdini/plugins/publish/validate_cop_output_node.py

```diff
@@ -71,8 +58,9 @@ def get_invalid(cls, instance):
         # the isinstance check above should be stricter than this category
         if output_node.type().category().name() != "Cop2":
```

Comment:
I find this confusing. 
sudo code.
```
if the instance not instance of hou.CopNode: 
    print its category type name.  (1)
    mark output node as invalid. 
if the category type name is not Cop2: 
    print that node is not of category Cop2 
    mark output node as invalid.
```
Hypothetically, what if the first condition was valid and at the same time the second one was invalid ? 
then the printed value at (1) will be wrong.

Reply:
Sorry, I don't get what you're saying. As the comment in the code describes - this check _should_ be stricter and the previous one more generic. According to that comment then the first _may_ be valid but be disallowed here. I do agree that it's odd that this directly raises an error instead of logging and returning. It should still behave 'similarly'. So with that, I agree.